.gitlab/build_lassen.yml

```diff
@@ -0,0 +1,45 @@
+####
+# This is the share configuration of jobs for lassen
+.on_lassen:
+  variables:
+  tags:
+    - shell
+    - lassen
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /_lnone/ || $ON_LASSEN == "OFF"' #run except if ...
+      when: never
+    - when: on_success
+
+####
+# Load required CUDA module
+.with_cuda:
+  before_script:
+    - module load cuda/10.1.243
```

Comment:
>     - module load cuda/10.1.243 

Is this required? Typically, we include all necessary information within our host-configs so that we don't need to depend on modules.

Reply:
It is not required but you cannot rely on host-configs for the cuda module.  That module has to be loaded into the environment or it fails at runtime.  10.1.243 is currently the default but I left it here because when we update to cuda 11 we can just replace the number here.